### ISSUE TITLE
Keyboard tab focus is not moving to 'Close(x)' button in the 'Find and Replace' dialog, while navigating using keyboard 'TAB

### ISSUE THREAD

**Issue by lopapraharaj** [tinymce/tinymce#7277]
Keyboard tab focus is not moving to 'Close(x)' button in the 'Find and Replace' dialog, while navigating using keyboard 'TAB

Steps:

Navigate to the rich editor text tool bar using short cut (Alt+F10) and navigate to the 'Find and Replace' button using arrows and press enter using keyboard.
'Find and Replace' dialog will be displayed and navigate to the 'Close(x)' button using keyboard tab key and verify keyboard focus is moving or not

Actual Result:

Keyboard tab focus is not moving to 'Close(x)' button in the 'Find and Replace' dialog, while navigating using keyboard 'TAB' key.

Ex: Once the dialog is displayed, keyboard tab focus is moving to 'Find' edit field->Replace with->Match Case->Whole Words->Find->Replace->Replace All->Prev->Next->Find…etc.

 

Expected Result:

Keyboard tab focus should move to 'Close(x)' button in the 'Find and Replace' dialog, while navigating using keyboard 'TAB' key.

Ex: Once the dialog is displayed, keyboard tab focus should move to Close(x) button->'Find' edit field->Replace with->Match Case->Whole Words->Find->Replace->Replace All->Prev->Next->Close(x)->Find…etc.

 

Ensure that:

Once keyboard tab focus is moving to 'Close(x)' button in the dialog, on invoking the 'Close(x)' button dialog should get closed and focus should be rich text editor.

Browser/OS

Test Environment:

OS: Windows 10 Version 20H2

Browser: Edge Version 87.0.664.60, Chrome Version 87.0.4280.88

**Comment by TheSpyder** [comment]
Fixed in #8481 which will be in the 6.4 release
